sdk/src/transaction/sanitized.rs

```diff
@@ -51,15 +50,15 @@ impl SanitizedTransaction {
         tx: VersionedTransaction,
         message_hash: Hash,
         is_simple_vote_tx: Option<bool>,
-        address_loader: impl Fn(&v0::Message) -> Result<LoadedAddresses>,
+        address_loader: impl FnOnce(&[MessageAddressTableLookup]) -> Result<LoadedAddresses>,
```

Comment:
nit: Why does this need to be a `FnOnce`?

Reply:
Hmm it doesn't, I'll update, thanks!